.github/ISSUE_TEMPLATE/performance_issue.md

```diff
@@ -28,11 +28,19 @@ An example of this is
 A link to either profile.zip *Chromium only* or a firefox profiler *Firefox only*.
 
 <!--
-Please add a version of the browser you are using. 
-If you don't know how to get your browser/darkreader version please search it up online.
+  Please add a version of the browser you are using. 
+  If you don't know how to get your browser/darkreader version please search it up online.
 -->
 **System info:**
 
 - OS: [e.g. Windows, MacOS, Linux]
 - Browser: [e.g. chrome, safari]
 - Dark Reader Version: [e.g. 4.9.9]
+
+<!--
+  Did this performance occur with DarkReader off?
```

Comment:
"Do you have this performance issue with Dark Reader disabled?"
Sounds more natural to me.

Reply:
Done: ad48ea3